index.html

```diff
@@ -594,102 +638,264 @@ <h3><dfn>MonetizationStartEventInit</dfn> dictionary</h3>
               <dt><dfn>requestId</dfn> member</dt>
               <dd>
                 The Monetization ID (UUID V4) generated by the browser (see
-                <a href="#flow">Flow</a>).
+                [[[#flow]]]).
               </dd>
             </dl>
           </section>
         </section>
+
         <section
-          data-dfn-for="MonetizationProgressEvent"
-          data-link-for="MonetizationProgressEvent"
+          data-dfn-for="MonetizationStopEvent"
         >
+          <h2><dfn>MonetizationStopEvent</dfn> interface</h2>
+          <pre class="idl">
+            [SecureContext, Exposed=Window]
+            interface MonetizationStopEvent : Event {
```

Comment:
Our implementation and the example code in the explainer (in this repo) is using an event with a detail field (CustomEvent) ...